accord-core/src/main/java/accord/utils/AccordConfig.java

```diff
@@ -0,0 +1,11 @@
+package accord.utils;
+
+public class AccordConfig {
+    public long progress_log_scheduler_delay_in_ms = 200L;
+
+    public AccordConfig(long progress_log_scheduler_delay_in_ms) {
+        this.progress_log_scheduler_delay_in_ms = progress_log_scheduler_delay_in_ms;
+    }
```

Comment:
this constructor isn't something we can maintain, as we add more and more configs this will become an issue